service/src/main/java/org/databiosphere/workspacedataservice/dao/RecordDao.java

```diff
@@ -1024,4 +1024,14 @@ public void deleteRecordType(UUID instanceId, RecordType recordType) {
       throw e;
     }
   }
+
+  public void deleteAttribute(UUID instanceId, RecordType recordType, String attribute) {
+    namedTemplate
+        .getJdbcTemplate()
+        .update(
```

Comment:
Can you use the varargs update here?  I am guessing it's safer than doing string interpolation:
```suggestion
        .update("alter table ? drop column ?", getQualifiedTableName(recordType, instanceId), quote(SqlUtils.validateSqlString(attribute, ATTRIBUTE)))
```

Reply:
I can try that. I was following the example of other methods in this class.

Reply:
SQL doesn't accept bind parameters in certain positions - I believe table names and column names do not support bind params. I agree with Josh that we should _always_ prefer params instead of string interpolation - but I don't think it'll work here. That said, give it a try, and if you can get it to work, great!

Reply:
`getQualifiedTableName` includes quotes, which I'm guessing would interfere with using it as a parameter.

I'm wondering how to pass both the schema and table name. I tried:
```
.update(
            "alter table ?.? drop column ?",
            instanceId.toString(),
            recordType.getName(),
            SqlUtils.validateSqlString(attribute, ATTRIBUTE));
```

And got an error:
```
org.springframework.jdbc.BadSqlGrammarException: PreparedStatementCallback; bad SQL grammar
```

Reply:
I'd say don't hold off on blocking this PR based on this.  We have precedent to do it the way you did it...so if it just doesn't work for _reasons_ I'm ok with accepting it and moving on for the time being.